### PR TITLE
[cert-manager][docs] Add information about using your own CA

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Information about using your own certificates has been added to the module's FAQ.

## Why do we need it, and what problem does it solve?

This will improve the understanding of the documentation.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
 
```changes
section: docs
type: chore
summary: Minor documentation updates of the node-manager module.
impact_level: low
```

### CODE REVIEW

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
   EOF
   ```
 
+## Как использовать свой или промежуточный CA для заказа сертификатов?
+
+Для использования собственного или промежуточного CA:
+
+- Сгенерируйте сертификат:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:189]:
```suggestion
- Сгенерируйте сертификат (при необходимости):
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
...
+
+  ```shell
+  openssl genrsa -out rootCAKey.pem 2048
+  openssl req -x509 -sha256 -new -nodes -key rootCAKey.pem -days 3650 -out rootCACert.pem
+  ```
+- Создайте секрет в пространстве имён `d8-cert-manager` и добавьте в него закодированные в base64 файлы сертификата:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:195]:
```suggestion
- В пространстве имён `d8-cert-manager` создайте секрет, содержащий данные файлов сертификатов.

  Пример создания секрета с помощью команды kubectl:
  ###shell
  kubectl create secret tls internal-ca-key-pair -n d8-cert-manager --key="rootCAKey.pem" --cert="rootCACert.pem"
  ###

  Пример создания секрета из YAML-файла (содержимое файлов сертификатов должно быть закодировано в Base64):
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
...
+
+  ```yaml
+  apiVersion: v1
+  data:
+    tls.crt: <cat rootCACert.pem | base64 -w0>
+    tls.key: <cat rootCAKey.pem | base64 -w0>
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:201]:
```suggestion
    tls.crt: <результат команды `cat rootCACert.pem | base64 -w0`>
    tls.key: <результат команды `cat rootCAKey.pem | base64 -w0`>
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
...
+  kind: Secret
+  metadata:
+    name: internal-ca-key-pair
+    namespace: d8-cert-manager
+  type: Opaque
+  ```
+
+  Название секрета может быть любым.
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:209]:
```suggestion
  Имя секрета может быть любым.
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 5]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
...
+  apiVersion: cert-manager.io/v1
+  kind: ClusterIssuer
+  metadata:
+    name: inter-ca
+  spec:
+    ca:
+      secretName: internal-ca-key-pair    # Имя созданного секрета.
+  ```
+
+  Название также может быть любым.
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:223]:
```suggestion
  Имя `ClusterIssuer` также может быть любым.
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 6]
```diff
@@ -182,6 +182,60 @@ CAA record does not match issuer
...
+
+Теперь можно использовать созданный ClusterIssuer для получения сертификатов. Например, для получения сертификатов для всех компонентов Deckhouse отредактируйте глобальную конфигурацию (`kubectl edit mc global`), указав секцию для заказа сертификатов:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:225]:
```suggestion
Теперь можно использовать созданный ClusterIssuer для получения сертификатов для всех компонентов Deckhouse или конкретного компонента.

Например, чтобы использовать ClusterIssuer для получения сертификатов для всех компонентов Deckhouse, укажите его имя в глобальном параметре [clusterIssuerName](../../deckhouse-configure-global.html#parameters-modules-https-certmanager-clusterissuername) (`kubectl edit mc global`):
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 7]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
   EOF
   ```
 
+## Как использовать свой или промежуточный CA для заказа сертификатов?
+
+Для использования собственного или промежуточного CA:
+
+- Сгенерируйте сертификат (при необходимости):
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:186]:
```suggestion
## How do I use a custom or interim CA to issue certificates?

Follow the steps below to use a custom or interim CA:

- Generate a certificate (if necessary):
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 8]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+
+  ```shell
+  openssl genrsa -out rootCAKey.pem 2048
+  openssl req -x509 -sha256 -new -nodes -key rootCAKey.pem -days 3650 -out rootCACert.pem
+  ```
+- В пространстве имён `d8-cert-manager` создайте секрет, содержащий данные файлов сертификатов.
+
+  Пример создания секрета с помощью команды kubectl:
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:194]:
```suggestion
- In the `d8-cert-manager` namespace, create a secret containing certificate file data.

  An example of creating a secret with kubectl:
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 9]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+  ```shell
+  kubectl create secret tls internal-ca-key-pair -n d8-cert-manager --key="rootCAKey.pem" --cert="rootCACert.pem"
+  ```
+
+  Пример создания секрета из YAML-файла (содержимое файлов сертификатов должно быть закодировано в Base64):
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:199]:
```suggestion
  An example of creating a secret from a YAML file (the contents of the certificate files must be Base64-encoded):
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 10]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+
+  ```yaml
+  apiVersion: v1
+  data:
+    tls.crt: <результат команды `cat rootCACert.pem | base64 -w0`>
+    tls.key: <результат команды `cat rootCAKey.pem | base64 -w0`>
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:205]:
```suggestion
    tls.crt: <OUTPUT OF `cat rootCACert.pem | base64 -w0`>
    tls.key: <OUTPUT OF `cat rootCAKey.pem | base64 -w0`>
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 11]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+  kind: Secret
+  metadata:
+    name: internal-ca-key-pair
+    namespace: d8-cert-manager
+  type: Opaque
+  ```
+
+  Имя секрета может быть любым.
+
+- Создайте ClusterIssuer из созданного секрета:
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:215]:
```suggestion
  You can use any name you like for the secret.

- Create a ClusterIssuer using the secret you created earlier:
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 12]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+
+  ```yaml
+  apiVersion: cert-manager.io/v1
+  kind: ClusterIssuer
+  metadata:
+    name: inter-ca
+  spec:
+    ca:
+      secretName: internal-ca-key-pair    # Имя созданного секрета.
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:224]:
```suggestion
      secretName: internal-ca-key-pair    # Name of the secret you created earlier.
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 13]
```diff
@@ -179,6 +179,70 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+  ```
+
+  Имя ClusterIssuer также может быть любым.
+
+Теперь можно использовать созданный ClusterIssuer для получения сертификатов для всех компонентов Deckhouse или конкретного компонента.
+
+Например, чтобы использовать ClusterIssuer для получения сертификатов для всех компонентов Deckhouse, укажите его имя в глобальном параметре [clusterIssuerName](../../deckhouse-configure-global.html#parameters-modules-https-certmanager-clusterissuername) (`kubectl edit mc global`):
```
Comment (kirkonru) [inline modules/101-cert-manager/docs/FAQ.md:231]:
```suggestion
  You can use any name as your ClusterIssuer name.

You can now use the created ClusterIssuer to issue certificates for all Deckhouse components or a particular component.

For example, to issue certificates for all Deckhouse components, specify the ClusterIssuer name in the [clusterIssuerName](../../deckhouse-configure-global.html#parameters-modules-https-certmanager-clusterissuername) global parameter (`kubectl edit mc global`):
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6641](https://github.com/deckhouse/deckhouse/pull/6641) to the branch [release-1.54](https://github.com/deckhouse/deckhouse/tree/release-1.54) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6643](https://github.com/deckhouse/deckhouse/pull/6643) to the branch [release-1.55](https://github.com/deckhouse/deckhouse/tree/release-1.55) successful!

**Pull request merged**